src/NzbDrone.Common/Extensions/StringExtensions.cs

```diff
@@ -22,9 +22,14 @@ public static object NullSafe(this object target)
             return "[NULL]";
         }
 
+        public static string FirstCharToLower(this string input)
+        {
+            return input.First().ToString().ToLower() + input.Substring(1);
```

Comment:
Use ToLowerInvariant, otherwise you'll get bad conversions in some languages.
btw. char.ToLowerInvariant(input.First()) would work too.

Same at line 32.